lib/tasks/dev.rake

```diff
@@ -0,0 +1,89 @@
+desc "Fill the database tables with some sample data"
+task({ :sample_data => :environment}) do
+  
+  User.destroy_all
+  Loan.destroy_all
+  Adjustment.destroy_all
+
+  # Create Sample Users
+    #  email                  :string           default(""), not null
+    #  encrypted_password     :string           default(""), not null
+  email = ["alice@example.com", "bob@example.com", "carol@example.com"]
+
+  3.times do |count|
+    user = User.new
+    user.email = email.at(count)
+    user.password = "password"
+    user.save
+  end
```

Comment:
This is perfectly functional, but there are a couple of techniques that might be a bit more idiomatic. Consider the following:

```ruby
  emails = ["alice@example.com", "bob@example.com", "carol@example.com"]

  emails.each do |email|
    user = User.new
    user.email = email
    user.password = "password"
    user.save
  end
```

Or:

```ruby
  emails = ["alice@example.com", "bob@example.com", "carol@example.com"]

  emails.each do |email|
    User.create(
      email: email,
      password: "password"
    )
  end
```

Or:

```ruby
  user_hashes = [
    { email: "alice@example.com", password: "password" },
    { email: "bob@example.com", password: "password" },
    { email: "carol@example.com", password: "password }
  ]

  user_hashes.each do |user_hash|
    User.create(user_hash)
  end
```

Or:

```ruby
  user_hashes = [
    { email: "alice@example.com", password: "password" },
    { email: "bob@example.com", password: "password" },
    { email: "carol@example.com", password: "password }
  ]

  User.create(user_hashes)
```

Reply:
Thanks for the suggestions! These all make sense and are a bit more robust and succinct.  I'll make this update.